unity-renderer/Assets/Scripts/MainScripts/DCL/DataStore/DataStore.cs

```diff
@@ -152,7 +153,18 @@ public class DataStore_FeatureFlag
 
         public class DataStore_Camera
         {
-            public readonly BaseVariable<Quaternion> rotation =  new BaseVariable<Quaternion>();
+            public readonly BaseVariable<Quaternion> rotation = new BaseVariable<Quaternion>();
+        }
+
+        public class DataStore_SkyboxConfig
+        {
+            public BaseVariable<bool> objectUpdated = new BaseVariable<bool>(false);
+            public BaseVariable<bool> useProceduralSkybox = new BaseVariable<bool>(false);
+            public BaseVariable<string> configToLoad = new BaseVariable<string>("Generic Skybox");
+            public BaseVariable<float> minutesPerSecond = new BaseVariable<float>(60);
+            public BaseVariable<bool> pauseTime = new BaseVariable<bool>(false);
+            public bool jumpTime = false;
+            public BaseVariable<float> jumpToTime = new BaseVariable<float>(0);
```

Comment:
Why do we have `jumpTime` and `jumpToTime`? Please change the name or remove the variable if its not used.

Reply:
- jumpToTime (BaseVariable<float>) is used when we have to jump to a specific time of the day... can provide float value of time.
- jumpTime (bool) is a flag after which "jumpToTime" is processed. 

Bool is required to assure if the value in "jumpToTime" should be applied or not, else it can lead to some bugs.